django_elasticsearch_dsl/documents.py

```diff
@@ -124,6 +148,12 @@ def to_field(cls, field_name, model_field):
     def bulk(self, actions, **kwargs):
         return bulk(client=self._get_connection(), actions=actions, **kwargs)
 
+    def parallel_bulk(self, actions, **kwargs):
+        deque(parallel_bulk(client=self._get_connection(), actions=actions, **kwargs), maxlen=0)
```

Comment:
@mjl Can you add the `chunk_size` parameter here which will consist the value of `queryset_pagination`?